package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Sprint-Challenge-RDBMS",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/jamiegoodnight/Sprint-Challenge-RDBMS.git",
+  "author": "Jamie Goodnight <goodnightlambda@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "server": "nodemon"
+  },
+  "dependencies": {
+    "express": "^4.16.4",
+    "knex": "^0.16.3",
+    "knex-cleaner": "^1.1.4",
+    "nodemon": "^1.18.10",
+    "sqlite3": "^4.0.6"
+  }
```

Comment:
You can also put knex-cleaner and nodemon in dev-dependencies.  Just no node dependencies 😝. Obviously this works too :)